packages/styles/src/components/card.scss

```
@@ -10,26 +10,79 @@
 @use './../mixins/icons' as icons-mx;
 @use './../variables/icons';
 @use './../variables/animation';
+@use '../mixins/utilities' as utilities-mx;
 
 @use './../functions/tokens';
 @use './../tokens/utilities';
+@use './../tokens/components';
 
-tokens.$default-map: utilities.$post-spacing;
+tokens.$default-map: components.$post-cards;
 
 .card-group {
   margin-top: commons.$border-width;
   margin-left: commons.$border-width;
-  box-shadow: commons.$box-shadow-lg;
+  box-shadow: tokens.get('card-interactive-elevation');
+  border-radius: tokens.get('card-border-radius');
 }
 
 .card {
-  transition: box-shadow 0.25s;
-  border: 0;
-  border-radius: 0;
-  // Outline with card-group margin trick prevents double/jumping (on hover) borders
-  outline: 1px solid card.$card-border-color;
-  box-shadow: commons.$box-shadow-lg;
+  transition: all 250ms;
   color: card.$card-color;
+  border-radius: tokens.get('card-border-radius');
+  background-color: tokens.get('card-enabled-bg1');
+
+  .palette-alternate & {
+    background-color: tokens.get('card-enabled-bg2');
+  }
+
+  // Add an elevation only if card is interactive
+  &.interactive-card {
+    border: 0;
+    box-shadow: tokens.get('card-interactive-elevation');
+
+    &:hover {
+      background-color: tokens.get('card-hover-bg');
+      color: tokens.get('card-hover-fg');
+    }
+
+    a:first-of-type,
+    button:first-of-type {
+      position: unset;
+      outline-style: none !important;
+
+      &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        cursor: pointer;
+        border-radius: tokens.get('card-border-radius', components.$post-cards);
+      }
+
+      @include utilities-mx.focus-style('::before');
+    }
+  }
+
+  // Add a border if card has no interactive element
+  &:not(.interactive-card) {
+    border: 1px solid tokens.get('card-non-interactive-stroke');
+  }
+
+  .card-header:first-child,
+  .list-group:first-child,
+  .card-img-top {
+    border-top-left-radius: tokens.get('card-border-radius');
+    border-top-right-radius: tokens.get('card-border-radius');
```

Can you replace all `top`, `bottom`, `right`, and `left` properties with their `start`, `end` equivalent? 

```suggestion
    border-start-start-radius: tokens.get('card-border-radius');
    border-start-end-radius: tokens.get('card-border-radius');
```